### PR TITLE
(1152) Apply -  Additional Circumstances change

### PR DESCRIPTION
**Pull request by richpjames**
# Context
We need to move the 'Additional circumstances' question from the Case notes page to it's own seperate page after the Arson page. 

[Trello card 1](https://trello.com/c/owJ3Zzk6/1152-add-additional-circumstances-screen-into-further-placement-considerations)
[Trello card 2](https://trello.com/c/ntrUXIk0/1174-prison-information)

# Changes in this PR
1. Some minor bug fixes to #706 #710 
2. Add the new additional circumstances page
3. Update the flow through the application so the user can go to the new page
4. Remove the old additional circumstances question from case notes page
 
### Before
Apply -- allows completion of the form (3)

### After
Apply -- allows completion of the form (5)
Apply -- allows completion of the form (4)

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: server/form-pages/apply/risk-and-need-factors/further-considerations/additionalCircumstances.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        this.body,
+      ),
+    }
+  }
+
+  errors() {
+    const errors: TaskListErrors<this> = {}
+
+    if (!this.body.additionalCircumstances) {
+      errors.additionalCircumstances = `You must specify if ${this.application.person.name} there are any additional circumstances that have helped ${this.application.person.name} ${this.questionPredicates.additionalCircumstances}`
```
Comment (pezholio) [inline server/form-pages/apply/risk-and-need-factors/further-considerations/additionalCircumstances.ts:54]: Would probably replace the last `${this.application.person.name}` with `them`, as it reads a bit better.

File: server/views/applications/pages/further-considerations/additional-circumstances.njk [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{% block questions %}
+
+    <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">Further placement considerations</span>
+        {{ page.title }}
+    </h1>
+
+    <p>This information is used to match a person with a suitable room.</p>
+
+    {% for key, value in page.questions %}
```
Comment (pezholio) [inline server/views/applications/pages/further-considerations/additional-circumstances.njk:12]: Do we need a loope here, as it's only one question?

**Review by pezholio — APPROVED** [review]
LGTM - just a few minor things

File: server/form-pages/apply/risk-and-need-factors/further-considerations/additionalCircumstances.ts (hunk 1 above)
Comment (richpjames) [reply server/form-pages/apply/risk-and-need-factors/further-considerations/additionalCircumstances.ts:54]: Good spot

**Pull request merged**